devops/jobs/CreateSandboxCI.groovy

```diff
@@ -21,7 +21,7 @@ import static org.edx.jenkins.dsl.DevopsConstants.common_read_permissions
 class CreateSandboxCI {
   public static def jobs = { dslFactory, extraVars ->
 
-    ["Hourly", "Daily", "MastersWeekly"].each { type ->
+    ["Hourly", "Daily", "MastersWeekly", "ProspectusBiHourly"].each { type ->
```

Comment:
I know it deviates from an established pattern, but I bend over backwards whenever possible to avoid words like "bimonthly" because they induce the moment of panic of "do I actually know whether that's half as often or twice as often"? I can't be the only one. Can we use `ProspectusEveryTwoHours` instead?